cartography/intel/aws/kms.py

```diff
@@ -76,11 +76,10 @@ def get_policy(key: Dict, client: botocore.client.BaseClient) -> Any:
     try:
         policy = client.get_key_policy(KeyId=key["KeyId"], PolicyName="default")
     except ClientError as e:
-        policy = None
         if e.response["Error"]["Code"] == "AccessDeniedException":
-            logger.warning(
-                f"kms:get_key_policy on key id {key['KeyId']} failed with AccessDeniedException; continuing sync.",
-                exc_info=True,
+            policy = None
+            logger.info(
+                f"Skipping KMS key {key['KeyId']} policy due to AccessDenied; will use default policy values"
```

Comment:
I'm flexible here but I don't think we should change the logged message here. If we don't have access, I'd like it to be easy to trace and not have it be buried as an INFO log.